index.html

```diff
@@ -0,0 +1,130 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <title>Elearnig Website 2022, 2023</title>
+    <link rel="stylesheet" href="./style.css" >
+    <script defer src="./script.js"></script>
+    <script src="https://kit.fontawesome.com/10729b3e67.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+  </head>
+  <body>
+      <header>
```

Comment:
Great job @Yordinia 🤝

- [ ] Kindly reduce the second navbar's height so it can replicate the design given.
- [ ] You may also watch and align your navbar elements correctly as provided in the design.
- Your design
![your-navbar](https://user-images.githubusercontent.com/63917725/205317587-ddbc2ee6-becb-43c7-8317-afe3d983c60d.PNG).
- Given design
![Given navbar](https://user-images.githubusercontent.com/63917725/205317757-92d2d6c9-ca89-45ba-a13d-a552404a01c9.PNG)